openedx/tests/completion_integration/test_handlers.py

```diff
@@ -121,7 +121,7 @@ def test_handler_skips_discussion_block(self):
         self.assertFalse(completion.exists())
 
     def test_signal_calls_handler(self):
-        with patch('completion.handlers.scorable_block_completion') as mock_handler:
+        with patch('completion.handlers.BlockCompletion.objects.submit_completion') as mock_handler:
```

Comment:
An assertion below wasn't valid in 1.0.1, upgrading revealed that this mock needed to be moved.